src/styles/_catalog.scss

```diff
@@ -1,22 +1,36 @@
-.catalog {
-  display: flex;
-  flex-wrap: wrap;
+$width: 200px;
 
-  max-width: 944px;
-  margin: 0 auto;
+.catalog {
+  display: grid;
+  justify-content: center;
```

Comment:
It's the unusual usage of this rule outside of flex. Not all browsers support this approach.
Could you remove it and use `margin: auto` and `max-width` (defined for every media query)